ote/src/cljs/ote/views/pre_notices/pre_notice.cljs

```diff
@@ -69,79 +70,86 @@
+     ;  :type      :string
+     ;  :update!   nil
+     ;  :disabled? true}
+     ; (::db-common/postal_code (::t-operator/visiting-address operator))] ;Commented out because addresses are not currently in use
+
+     ;[form-fields/field
+     ; {:label     (tr [:field-labels ::db-common/post_office])
+     ;  :name      ::db-common/post_office
+     ;  :type      :string
+     ;  :update!   nil
+     ;  :disabled? true}
+     ; (::db-common/post_office (::t-operator/visiting-address operator))] ;Commented out because addresses are not currently in use
+
+     [:div (stylefy/use-style sbase/item-list-row-margin)
       [form-fields/field
-       {:label     (tr [:field-labels ::db-common/postal_code])
-        :name      ::db-common/postal_code
-        :type      :string
-        :update!   nil
-        :disabled? true}
-       (::db-common/postal_code (::t-operator/visiting-address operator))]
-      [form-fields/field
-       {:label     (tr [:field-labels ::db-common/post_office])
-        :name      ::db-common/post_office
-        :type      :string
-        :update!   nil
-        :disabled? true}
-       (::db-common/post_office (::t-operator/visiting-address operator))]]
-     [:div.col-xs-8.col-sm-6.col-md-6
-      [form-fields/field
-       {:label     (tr [:field-labels ::t-operator/homepage])
-        :name      ::t-operator/business-id
-        :type      :string
-        :update!   nil
-        :disabled? true}
-       (::t-operator/homepage operator)]
-      [form-fields/field
-       {:label     (tr [:field-labels ::t-operator/phone])
-        :name      ::t-operator/phone
-        :type      :string
-        :update!   nil
-        :disabled? true}
-       (::t-operator/phone operator)]
-      [form-fields/field
-       {:label     (tr [:field-labels ::t-operator/gsm])
-        :name      ::t-operator/gsm
-        :type      :string
-        :update!   nil
+       {:label (tr [:field-labels ::t-operator/phone])
+        :name ::t-operator/phone
+        :type :string
+        :update! nil
         :disabled? true}
-       (::t-operator/gsm operator)]
+       (::t-operator/phone operator)]]
+
+     [form-fields/field
+      {:label (tr [:field-labels ::t-operator/gsm])
+       :name ::t-operator/gsm
+       :type :string
+       :update! nil
+       :disabled? true}
+      (::t-operator/gsm operator)]]
+
+    [:div (stylefy/use-style sbase/item-list-container)
+     [:div (stylefy/use-style sbase/item-list-row-margin)
       [form-fields/field
-       {:label     (tr [:field-labels ::t-operator/email])
-        :name      ::t-operator/email
-        :type      :string
-        :update!   nil
+       {:label (tr [:field-labels ::t-operator/homepage])
+        :name ::t-operator/business-id
+        :type :string
+        :update! nil
         :disabled? true}
-       (::t-operator/email operator)]]]]])
+       (::t-operator/homepage operator)]]
+
+     [form-fields/field
+      {:label (tr [:field-labels ::t-operator/email])
```

Comment:
You should move type at the beginning here as well.

Reply:
Will push a commit next